extras/rya.geoindexing/geo.mongo/pom.xml

```diff
@@ -0,0 +1,29 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.apache.rya</groupId>
+    <artifactId>rya.geoindexing</artifactId>
+    <version>3.2.11-incubating-SNAPSHOT</version>
+  </parent>
+  <artifactId>rya.geomongo</artifactId>
+  <name>Geo_indexing_for_mongo</name>
+  <description>Implementation of a geospatial indexing for mongo DB backed Rya</description>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <geotools.version>16.0</geotools.version>
+  </properties>
+  <dependencies>
+  	<dependency>
+  		<groupId>org.apache.rya</groupId>
+  		<artifactId>rya.geoCommon</artifactId>
+  		<version>3.2.11-incubating-SNAPSHOT</version>
```

Comment:
This one line fixed all the old names.  
find . -name \*.xml  -exec sed -i 's/rya.geoCommon/geo.common/g'  {} \; -ls